.github/workflows/ci-cd-workflow.yml

```diff
@@ -259,7 +256,40 @@ jobs:
         pip install --upgrade pip wheel
         pip install -e .[dev]
     - name: Create package
-      run: python setup.py sdist bdist_wheel --universal
+      run: |
+        python setup.py sdist bdist_wheel --universal
+        twine check dist/*
+    - name: Check package files
+      run: |
+        for f in README.rst LICENSE CHANGELOG.rst
```

Comment:
can we add a check for `data/metric_conversions.csv'? That should be the only data file that needs to be bundled

Reply:
Scratch that. I can't find a reference anywhere in the code for that file so it probably isn't needed

Reply:
ye that got moved to openscm-units